xarray/backends/api.py

```diff
@@ -147,7 +147,8 @@ def _get_lock(engine, scheduler, format, path_or_file):
 def open_dataset(filename_or_obj, group=None, decode_cf=True,
                  mask_and_scale=True, decode_times=True, autoclose=False,
                  concat_characters=True, decode_coords=True, engine=None,
-                 chunks=None, lock=None, cache=None, drop_variables=None):
+                 chunks=None, lock=None, cache=None, drop_variables=None,
+                 backend_kwargs={}):
```

Comment:
Also, it's generally considered bad form to use mutable arguments like dictionaries as default values. Better to use `backend_kwargs=None` and then in the body of the function:
```
if backend_kwargs is None:
    backend_kwargs = {}
```

Reply:
Done.